packages/moonstone/ExpandablePicker/ExpandablePicker.js

```diff
@@ -44,6 +44,14 @@ const ExpandablePickerBase = kind({
 		 */
 		children: PropTypes.node.isRequired,
 
+		/**
+		 * The `data-webos-voice-group-label` for ExpandableItem and Picker.
+		 *
+		 * @type {String}
+		 * @public
```

Comment:
This needs an `@memberof` or it will not document properly.  The parser has a problem with quoted property names.